stable/cluster-backup-chart/templates/hub-backup-pod.yaml

```diff
@@ -0,0 +1,165 @@
+# Copyright Contributors to the Open Cluster Management project
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  annotations:
+    "helm.sh/hook": pre-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+    policy.open-cluster-management.io/categories: PR.IP Information Protection Processes and Procedures
+    policy.open-cluster-management.io/controls: PR.IP-4 Backups of information are conducted, maintained, and tested
```

Comment:
Commas need to be removed otherwise UI will treat them as multiple controls

Reply:
done, replaced with 
```
    policy.open-cluster-management.io/controls: PR.IP-4 Backups of information are conducted maintained and tested

```